proto/path/qos/cometbft.proto

```diff
@@ -47,8 +47,10 @@ message CometBFTHealthResponse {
 
 // CometBFTBlockNumberResponse stores the latest block number from a `status` request
 // Reference: https://docs.cometbft.com/v1.0/spec/rpc/#status
-message CometBFTLatestBlockHeightResponse {
-    string latest_block_height_response = 1;
+message CometBFTStatusResponse {
```

Comment:
Wdyt of adding a TODO_FUTURE to use the actual external protobuf: https://github.com/cometbft/cometbft/blob/4226b0ea6ab4725ef807a16b86d6d24835bb45d4/rpc/core/types/responses.go#L103

Note: I'm not using `main` to make sure there's always something to reference.

Reply:
I just imported the package; I like not defining our own types if possible.